week1/Test.html

```diff
@@ -0,0 +1,46 @@
+<html>
+<head>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+</head>
+<body>
+Email <input id = 'email' type = 'text'/><br>
+Password <input id = 'password' type = 'password'/><br>
+<button id='login'>Sign Up</button>
+
+<script>
```

Comment:
@ilikealotgithub1234 My first feedback would be to indent everything properly. Atom.io has an auto indentation button... maybe google for it.

Next comment the JavaScript. You'll be essentially Rubber Ducking. I spot a few errors that I think you can solve by rubber ducking / commenting your work.